cli_reference/openshift_developer_cli/installing-odo.adoc

```diff
@@ -5,14 +5,15 @@ include::modules/common-attributes.adoc[]
 :context: installing-odo
 toc::[]
 
-The following section describes how to install `{odo-title}` on different platforms.
+The following section describes how to install `{odo-title}` on different platforms using the CLI.
+
+Currently, `{odo-title}` does not support installation in a restricted network environment.
 
 [NOTE]
 ====
-Currently, `{odo-title}` does not support installation in a restricted network environment.
+You can also find the URL to the latest binaries in the web console of your cluster. To access them using the web console; click the *?* icon in the upper-right corner of the window, and select *Command Line Tools* in the drop-down menu.
```

Comment:
* Suggest simplifying to something like "You can also download the latest binaries from the {product-title} web console." Or even combining into "You can also download the latest binaries from the {product-title} web console by clicking the *?* icon in the upper-right corner and select *Command Line Tools*."
* Use a comma instead of a semicolon in the second sentence.

Reply:
You can't really _download_ them this way. Clicking the ? icon in the upper-right corner and selecting Command Line Tools will lead user to the page with titles and URLs of different CLI tools' folders on the file server (odo, helm etc). And there, on the file server, they can download the actual binary file. 

Otherwise, good call, thank you! I've shortened the sentence and included it as a part of introduction in both modules.

Reply:
Fair enough!